microshift_troubleshooting/microshift-things-to-know.adoc

```diff
@@ -17,4 +17,8 @@ When such changes occur, some {microshift-short} components may stop functioning
 
 The threshold for clock changes is a time adjustment of greater than 10 seconds in either direction. Smaller drifts on regular time adjustments performed by the Network Time Protocol (NTP) service do not cause a restart.
 
-include::modules/microshift-certificate-lifetime.adoc[leveloffset=+1]
\ No newline at end of file
+include::modules/microshift-certificate-lifetime.adoc[leveloffset=+1]
+[role="_additional-resources"]
+.Additional resources
+
+* For more information about custom certificate authorities, see xref:../microshift_configuring/microshift_auth_security/microshift-custom-ca.adoc#microshift-custom-ca[Configuring custom certificate authorities].
```

Comment:
I think the link lead in sentence could be dropped as the heading is clear with regards to the custom element.